Tests/Unit/Tools/SmokeTests/QueueDriverConnectionSmokeTestTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Smartbox\Integration\FrameworkBundle\Tests\Unit\Tools\SmokeTests;
+
+use PhpAmqpLib\Exception\AMQPProtocolException;
+use PHPUnit\Framework\TestCase;
+use Smartbox\CoreBundle\Utils\SmokeTest\Output\SmokeTestOutput;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\QueueDriverInterface;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\QueueMessageInterface;
+use Smartbox\Integration\FrameworkBundle\Tools\SmokeTests\QueueDriverConnectionSmokeTest;
+
+/**
+ * @group queue_driver_smoke-test
+ */
+class QueueDriverConnectionSmokeTestTest extends TestCase
+{
+    /**
+     * Test that the smoke test can be test...ed
+     */
+    public function testHappyPath()
+    {
+        $queueDriver = $this->createMock(QueueDriverInterface::class);
+        $queueDriver
+            ->method('isConnected')
+            ->willReturn(true);
+        $queueDriver
+            ->method('send')
+            ->willReturn(true);
```

Comment:
I think in this case would be interesting to add how many times these methods are expected to be called

Reply:
Alright, I included the expect time.

Reply:
Hey this one was mine :F

Reply:
This comment applies also for the other test cases in this test

Reply:
Fixed